### ISSUE TITLE
Incorrect number of pages for Table

### ISSUE THREAD

**Issue by natabal** [nuagenetworks/vis-graphs#94]
The number of pages shows incorrectly as per below screenshot

<img width="352" alt="screen shot 2019-01-14 at 8 45 32 am" src="https://user-images.githubusercontent.com/4873481/51126832-f73f3400-17d8-11e9-93c9-e9170ca972dd.png">

**Comment by nxanil** [comment]
@natabal I looked at the library and seems like we cannot override the styling of the Footer.
Apart from this 1-4 of 4 means, first 4 records are displayed from the total of 4 records.

Suppose if we have a paging size is of 2, then the first page will be displayed as `1-2 of 4` and the second page will be displayed as `3-4 of 4`.
 
Along with this, we have a config named as "hidePagination" default to true, if we have only one page.
So this will hide the pagination if we have can display all the records in the single page. This config seems to be broken and I have created the PR: #98
